test/units/testsuite-64.sh

```diff
@@ -588,14 +588,17 @@ testcase_mdadm_basic() {
     udevadm wait --settle --timeout=30 "$raid_dev"
     mkfs.ext4 -L "$part_name" "$raid_dev"
     udevadm wait --settle --timeout=30 "${expected_symlinks[@]}"
-    # Disassemble the array
-    mdadm -v --stop "$raid_dev"
-    udevadm settle
-    helper_check_device_symlinks
-    # Reassemble it and check if all required symlinks exist
-    mdadm --assemble "$raid_dev" --name "$raid_name" -v
-    udevadm wait --settle --timeout=30 "${expected_symlinks[@]}"
-    helper_check_device_symlinks
+    for i in {0..9}; do
+        echo "Disassemble - reassemble loop, iteration #$i"
+        # Disassemble the array
+        mdadm -v --stop "$raid_dev"
+        udevadm settle
+        helper_check_device_symlinks
+        # Reassemble it and check if all required symlinks exist
+        mdadm --assemble "$raid_dev" --name "$raid_name" -v
+        udevadm wait --settle --timeout=30 "${expected_symlinks[@]}"
+        helper_check_device_symlinks
+    done
```

Comment:
Let's make the test harder; disassemble and reassemble 10 times without waits, then finally check the validity
```
for i in {0..9}; do
    mdadm --stop ...
    mdadm --assemble ...
done
udevadm wait ...
helper_check...
```